ksql-rest-app/src/test/resources/ksql-plan-schema/schema-final.json

```diff
@@ -0,0 +1,885 @@
+          "enum" : [ "tableAggregateV1" ],
+          "default" : "tableAggregateV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        },
+        "nonFuncColumnCount" : {
+          "type" : "integer"
+        },
+        "aggregations" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        }
+      },
+      "title" : "tableAggregateV1",
+      "required" : [ "@type", "properties", "source", "formats", "nonFuncColumnCount", "aggregations" ]
+    },
+    "TableFilter" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "tableFilterV1" ],
+          "default" : "tableFilterV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "filterExpression" : {
+          "type" : "string"
+        }
+      },
+      "title" : "tableFilterV1",
+      "required" : [ "@type", "properties", "source", "filterExpression" ]
+    },
+    "TableGroupBy" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "tableGroupByV1" ],
+          "default" : "tableGroupByV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        },
+        "groupByExpressions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        }
+      },
+      "title" : "tableGroupByV1",
+      "required" : [ "@type", "properties", "source", "formats", "groupByExpressions" ]
+    },
+    "TableMapValues" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "tableMapValuesV1" ],
+          "default" : "tableMapValuesV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "selectExpressions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "selectNodeName" : {
```

Comment:
maybe `selectType` would be a better name?

Reply:
I think we can drop this and generate the name from the context.